corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -419,8 +445,6 @@ def get_detail_column_infos(detail_type, detail, include_sort):
     This is not intented to be a widely used format
     just a packaging of column info into a form most convenient for rendering
     """
-    DetailColumnInfo = namedtuple('DetailColumnInfo',
```

Comment:
Should move this comment along with the `DetailColumnInfo` definition.

Reply:
moved this back to it's original place.